### ISSUE TITLE
[AAMAS'2021] MAPFAST: A Deep Algorithm Selector for Multi Agent Path Finding using Shortest Path Embeddings

### ISSUE THREAD

**Issue by Fernadoo** [Fernadoo/Papers_and_Refs#1]
http://www.ifaamas.org/Proceedings/aamas2021/pdfs/p1055.pdf

**Comment by Fernadoo** [comment]
The algorithm portfolio contains CBS, BCP, SAT. The selector is supposed to select the fastest algorithm among them for each problem instance.

**Comment by Fernadoo** [comment]
Apart from basic information, also embeds single-agent-shortest-path encoding.

Deep model tested:
1. Inception-based network
2. Inception-based network with multiple loss functions
3. XGBoost with only single-agent-shortest path encoding

**Comment by Fernadoo** [comment]
Quite a lot of experiments, but the idea is simple.
